### DISCUSSION TITLE
How to use astroplate theme with astro-theme-provider?

### DISCUSSION THREAD

**Discussion by ProfAndreaPollini** [astrolicious/astro-theme-provider#3] [Q&A]
Hi,
  I'm using this package in order to simplify the management of my personal website which uses astroplate. The problem is that astroplates refers to layouts using a typescript path alias, so for example in the 404.astro page we have a import of `@/layouts/Base.astro` which resolves to an astroplate subpath. When I integrate the theme in my own project I get a `Cannot find module  @/layouts/Base.astro`. How can I manage it? putting path aliases in my integration module doesn't solve the error. Any advice? Thanks a lot.

**Comment by BryceRussell** [comment]
> Hi, I'm using this package in order to simplify the management of my personal website which uses astroplate. The problem is that astroplates refers to layouts using a typescript path alias, so for example in the 404.astro page we have a import of `@/layouts/Base.astro` which resolves to an astroplate subpath. When I integrate the theme in my own project I get a `Cannot find module @/layouts/Base.astro`. How can I manage it? putting path aliases in my integration module doesn't solve the error. Any advice? Thanks a lot.

Hello 👋

Interesting! I have not done any testing with import aliases so I am not too sure why this is happening. What does your code look like? Is there a repo that I can check out?

**Comment by BryceRussell** [comment]
Hey, I am going to close this for now. There have been a lot of major upgrades to the project, and using import aliases should now be supported. If you have any problems, feel free to open up an issue and I can help you out!
